dataclassy/__init__.py

```diff
@@ -10,3 +10,17 @@
 
 # aliases intended for migration from dataclasses
 asdict, astuple = as_dict, as_tuple
+
+__all__ = (
```

Comment:
I'd like to split these into three lines, like the imports above. Also, `as_dict`/`as_tuple` are repeated instead of referencing the aliases.

Reply:
Thanks. Yep, Got it!